services/user-management/src/main/java/com/foodie/user/controllers/PermissionController.java

```diff
@@ -1,18 +1,82 @@
 package com.foodie.user.controllers;
 
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
+import com.foodie.FoodieBaseResponse;
+import com.foodie.user.model.Permission;
 import com.foodie.user.service.PermissionService;
 
+import jakarta.validation.Valid;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.Optional;
+import java.util.UUID;
+
 @Slf4j
 @RestController
 @CrossOrigin
 @RequestMapping("/api/v1/users/permissions")
 public class PermissionController {
     
     private PermissionService permissionService;
```

Comment:
This field can be final

Reply:
Making it final breaks the code.

Reply:
<img width="937" alt="Screenshot 2024-03-25 at 9 10 52 pm" src="https://github.com/deye9/foodie/assets/33951975/5d5617b7-8037-4520-965f-6f37f67adbec">
This should work

Reply:
Final actually works. BReaks my test though but I will fix the test

Reply:
If it breaks the test, then there's something else wrong with the code or the test

Reply:
Issue is with the test